website/blog/2020-12-19-lithium.md

```diff
@@ -0,0 +1,211 @@
+---
+author: Tomasz Godzik
+title: Metals v0.9.8 - Lithium
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We're happy to announce the release of Metals v0.9.8, which brings along a
+number of smaller improvements, new Scala 3 milestone support and a hot new
+feature.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">99</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">40</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">4</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">5</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">1</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/31?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text, Atom and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- Add in ability to export worksheet evaluations.
+- Scala 3.0.0-M2 support.
+
+## Worksheet evaluations
+
+Thanks to the great work by [ckipp01](https://github.com/ckipp01) users will now
+be able to export their evaluated worksheets using a new command,
+`copy-worksheet-output`, or `Metals: Copy worksheet output` in Visual Studio
+Code.
+
+![worksheet-export](https://i.imgur.com/PQmT2i3.gif)
+
+When using this command existing statements will be copied to your clipboard
+along with their evaluations included as comments below them. This will allow
+users to freely share their results with others.
+
+## Miscellaneous
+
+- Add in Scala Sources to standalone worksheets.
+- Fix auto-connect BSP server without explicit choice.
+- Make sure to not duplicate semanticdbOptions when using sbt BSP.
+- Ensure semanticdb is enabled with sbt BSP and Scala 3
+
+## Contributors
+
+Big thanks to everybody who contributed to this release or reported an issue!
+
+```
+$ git shortlog -sn --no-merges v0.9.7..v0.9.8
+Chris Kipp
+Tomasz Godzik
+Gabriele Petronella
+Victor Mikhaylov
+```
+
+## Merged PRs
+
+## [v0.9.8](https://github.com/scalameta/metals/tree/v0.9.8) (2020-12-18)
+
+[Full Changelog](https://github.com/scalameta/metals/compare/v0.9.7...v0.9.8)
+
+**Merged pull requests:**
+
```

Comment:
Not a huge deal, but this last minute pr doesn't seem to be in here https://github.com/scalameta/metals/pull/2333

Other than that, as always, great job putting then together! LGTM 🚀

Reply:
Fixed!!